backend/tests/v5/unit/processors/users.test.js

```diff
@@ -225,6 +225,11 @@ const testGenerateResetPasswordToken = () => {
 
 const testSignUp = () => {
 	describe('Sign up a user', () => {
+		beforeAll(() => {
```

Comment:
do you need this at before all if only one test needed it?

Could we put it inside the test instead?